### PR TITLE
feat: hypercore page.

### PR DESCRIPTION
**Pull request by billy-the-fish**
Overview page explaining the advantages of the Hyperstore

### CODE REVIEW

File: use-timescale/hypercore.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
+---
+title: Hypercore
+excerpt: The Timescale hybrid row-columnar storage engine for real-time analytics, powered by time-series data
+products: [cloud,]
+keywords: [hyperscore, hypertable, compression, row-columnar storage, hyperstore]
```
Comment (billy-the-fish) [inline use-timescale/hypercore.md]:
```suggestion
keywords: [hyperscore, hypertable, compression, row-columnar storage, hypercore]
```

File: use-timescale/hypercore.md [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+- Efficient handling of data updates and late-arriving data 
+- Streamlined data management
+
+Currently, much of this functionality is available as [compression][compression]. However, we are actively 
+updating our APIs and docs so you can more easily integrate the following features under the hypercore umbrella.
+
+Hypercore’s hybrid approach combines the benefits of row-oriented and column-oriented formats 
+in each Timescale Cloud service:
+
+- **Fast ingest with Rowstore**: new data is initially written to the rowstore, it is optimized for
```
Comment (atovpeko) [inline use-timescale/hypercore.md]: Why capitalize Rowstore and Columnstore?

Comment (atovpeko) [inline use-timescale/hypercore.md]:
```suggestion
- **Fast ingest with Rowstore**: new data is initially written to the rowstore, which is optimized for 
```

**Comment by billy-the-fish** [comment]
Hi @ramonguiu , are we good to publish this or would you like to hold back a bit.

**Comment by billy-the-fish** [comment]
I've moved the content from here to https://github.com/timescale/docs/pull/3642 so this is all wrapped up with API updates.

**Pull request closed without merging**